s/bindings/types.ts

```diff
@@ -4,16 +4,8 @@ export interface Binds {
 }
 
 export interface Bindings {
-
-	/** comments */
 	comment: string[]
-
-	/** mouse action bindings */
 	mouse: Binds
-
-	/** vector2 action bindings */
 	vector2: Binds
-
-	/** key action bindings */
 	key: Binds
 }
```

Comment:
removed these old comments. they were more useful when they were explaining the meaning of the emojis